<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbortDialog</class>
 <widget class="QDialog" name="AbortDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Abort</string>
  </property>
  <widget class="QLabel" name="abort_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Abort Acquisition?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="abort_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
